src/Microsoft.OData.Edm/Vocabularies/CapabilitiesVocabularies.xml

```diff
@@ -1,18 +1,260 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Schema xmlns="http://docs.oasis-open.org/odata/ns/edm" Namespace="Org.OData.Capabilities.V1" Alias="Capabilities">
+
+  <!-- Conformance Level -->
+
+  <Term Name="ConformanceLevel" Type="Capabilities.ConformanceLevelType" AppliesTo="EntityContainer" />

```

Comment:
This file should be the same to the one in oasis.org: http://docs.oasis-open.org/odata/odata/v4.0/errata02/os/complete/vocabularies/Org.OData.Capabilities.V1.xml